### PR TITLE
EN: Support Partner Translations

### PR DESCRIPTION
**Pull request by rebelclover**
* Update sp_010300.csv

* Translated first lines of Personality Change

* Translated Talk 1 Lines

You are not seeing things. They are really the same. As in Talk 0.

* Removing line breaks

In accordance with established rules, unless there's a good reason to keep them in.

* Translating talk 3

The lines being the same make it really easy. No joke.

* Additional Translations

### CODE REVIEW

**Comment by rebelclover** [comment]
It still seems to be tripping over files in some way.

**Comment by DabirSA** [comment]
Have you synced your EN with Arks-Layer:EN?

File: Dialogue/Support Partner/sp_020000.csv [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-TALK_FAV1#0,"""I am a Support Partner.<br>I am happy to help you, master."""
-TALK_FAV0#0,"""I am a Support Partner.<br>I am happy to help you, master."""
-PERSENT_FAV4#0,"""Master, this isn't much,<br>but I hope this gift is of<br>good use to you. Please take it."""
-PERSENT_FAV3#0,"""Please accept this.<br>It's a symbol of my gratitude."""
-PERSENT_FAV2#0,"""This is a gift from me.<br>Please accept it."""
+TALK_FAV4#0,"""I am a Support Partner. I am happy to help you, master."""
+TALK_FAV3#0,"""I am a Support Partner. I am happy to help you, master."""
+TALK_FAV2#0,"""I am a Support Partner. I am happy to help you, master."""
+TALK_FAV1#0,"""I am a Support Partner. I am happy to help you, master."""
+TALK_FAV0#0,"""I am a Support Partner. I am happy to help you, master."""
```
Comment (DabirSA) [inline Dialogue/Support Partner/sp_020000.csv]: Please leave the line breaks in for the five copies of this line.

File: Dialogue/Support Partner/sp_020000.csv [hunk 2]
```diff
@@ -28,11 +28,11 @@ PERSENT_FAV3#2,"""こ、これ、受け取ってくれます？<br>日頃のお
...
-TALK_FAV4#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV3#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV2#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV1#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV0#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
+TALK_FAV4#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV3#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV2#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV1#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV0#3,"""I do not need words between us. I will do everything for you."""
```
Comment (DabirSA) [inline Dialogue/Support Partner/sp_020000.csv]: This line is kind of awkward. Maybe substitute with "There is no need for words between us. I will do anything for you."

**Comment by alama** [comment]
I guess I will sync your branch, wait a bit

File: UI/ui_supportpartoner.csv [hunk 3]
```diff
@@ -80,14 +80,14 @@ PartnerChangeEquipError7#0,"""クラス変更制限のため<br>装備を変更
...
-PartnerChangeEquipError11#0,"""未鑑定のアイテムは<br>渡すことができません。"""
+PartnerChangeEquipError11#0,"""Unidentified items can not be given."""
 PartnerChangeEquipError12#0,"""Inventory is full. Cannot retrieve equipped item."""
 PartnerChangeSubclassSuccess#0,"""Subclass changed."""
 PartnerChangeSubclassError#0,"""Failed to change subclass for<br>Support Partner."""
 PartnerChangeSameSubclass#0,"""Your subclass is already $(0)."""
 StartCheckButton1#0,"""Depart"""
 StartCheckButton2#0,"""Save and depart"""
-Leave#0,"""\u3000（あと $(0)）"""
+Leave#0,"""\u3000（after $(0)）"""
```
Comment (alama) [inline UI/ui_supportpartoner.csv]: remove `\u3000` and replace it with single or double space

**Comment by rebelclover** [comment]
The changes have been implemented. For reference (and I was going to do this, but internet fell apart) There are guides on GitHub's docs for [Adding the upstream repo as a remote](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and for [Syncronising the fork](https://help.github.com/articles/syncing-a-fork/).

File: Dialogue/Support Partner/sp_010000.csv [hunk 4]
```diff
@@ -8,10 +8,10 @@ CHOICE_NOISY#0,"""Dense"""
 CHOICE_SHARP#0,"""Clumsy"""
 CHOICE_BOYISH#0,"""Boyish"""
 CHOICE_NONE#0,"""No change"""
-CHANGE_START#0,"""質問があります。<br>マスターは私の個性を<br>どのように思っているのでしょうか。"""
-CHANGE_PERSONAL#0,"""今の私は、個性に乏しいのでは<br>ないかと思えてきたのです。"""
-CHANGE_ASK#0,"""ゆえにマスター、質問します。<br>私にどのように、なってほしいですか？"""
-CHANGE_OK#0,"""了解しました。<br>マスターの望み通りに……"""
+CHANGE_START#0,"""I have a question. What do you think of my personality, master?"""
+CHANGE_PERSONAL#0,"""I think that I am scarce in personality."""
```
Comment (DabirSA) [inline Dialogue/Support Partner/sp_010000.csv]: "lacking" in personality rather than "scarce", maybe?

Comment (rebelclover) [reply Dialogue/Support Partner/sp_010000.csv]: Sounds like it flows better, personally.

**Comment by Aida-Enna** [comment]
For reference, it might be easier to hop on Discord and discuss this stuff in real time (if you'd like!)

**Comment by rebelclover** [comment]
There's been some changes made to the EN branch. I'm merging those changes to avoid another issue with files causing CI to fail.

**Comment by alama** [comment]
you really only need to merge upstream changes IF the GitHub said there is a merge conflict and need a human to resolve it

**Pull request merged**